clang/bindings/python/pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[build-system]
+requires = ["hatchling>=1.27", "hatch-vcs>=0.4"]
+build-backend = "hatchling.build"
+
+[project]
+name = "clang"
+description = "clang python bindings"
+readme = {file = "README.txt", content-type = "text/plain"}
+
+license = "Apache-2.0 WITH LLVM-exception"
+authors = [
+    { name = "LLVM" }
+]
+keywords = ["llvm", "clang", "libclang"]
+classifiers = [
+    "Intended Audience :: Developers",
+    "Development Status :: 5 - Production/Stable",
+    "Topic :: Software Development :: Compilers",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+]
+requires-python = ">=3.10"
+dynamic = ["version"]
+
+[project.urls]
+Homepage = "https://clang.llvm.org/"
+Download = "https://llvm.org/releases/download.html"
```

Comment:
This website seems super outdated, are you sure there isn't a better one?

Reply:
For the homepage maybe the website theme is a bit on the dated side? Looking at the C++ status page, it lists features that were added in the v21 release: https://clang.llvm.org/cxx_status.html

The downloads page looks like it could use updating… or perhaps it could just get pointed at the GitHub releases page for LLVM?

Reply:
There's a GitHub issue about the download.html page: https://github.com/llvm/llvm-project/issues/166196

It sounds like it maybe just needs a PR that adds the new releases to the page. I have a few other fixes for the releases site and was about to push a commit, but GitHub has a major outage right now with all "Git Operations".